pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-konvoy/enterprise/index.md

```diff
@@ -80,14 +80,22 @@ Your cluster comes preconfigured with a few different core addons that provide f
 
 <p class="message--warning"><strong>IMPORTANT:</strong>Ensure your <code>dkp</code> configuration references the management cluster where you want to run the upgrade by setting the <code>KUBECONFIG</code> environment variable, or using the <code>--kubeconfig</code> flag, <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">in accordance with Kubernetes conventions</a>.
 
-Replace `my-aws-cluster` with the name of the cluster.
+Upgrade the core addons in a cluster using the 'dkp upgrade addons' command specifying the cluster infrastructure (i.e. one of [aws, azure, preprovisioned]) and the name of the cluster.
```

Comment:
I am not sure about this, but it looks a bit odd in front-end with both the normal and the square brackets... Or maybe this was meant to be a link? 
```suggestion
Upgrade the core addons in a cluster using the 'dkp upgrade addons' command specifying the cluster infrastructure (either aws, azure, or preprovisioned) and the name of the cluster.
```

Reply:
That was either Craig or Casie that changed it to the brackets.  So I just committed it.  HA!